renderdoc/os/posix/linux/linux_stringio.cpp

```diff
@@ -595,8 +595,16 @@ rdcstr GetTempRootPath()
 
 rdcstr GetAppFolderFilename(const rdcstr &filename)
 {
-  passwd *pw = getpwuid(getuid());
-  const char *homedir = pw->pw_dir;
+  const char *homedir = getenv("HOME");
```

Comment:
I would rather implement this the other way around - if `getpwuid` returns NULL or the `pw_dir` is empty/NULL then fall back to looking at `$HOME`. That way the existing working behaviour is not changed.

Reply:
Sounds good to me.

Reply:
Made the change.